lichess-bot.py

```diff
@@ -892,7 +893,8 @@ def intro():
 def start_lichess_bot():
     parser = argparse.ArgumentParser(description="Play on Lichess with a bot")
     parser.add_argument("-u", action="store_true", help="Upgrade your account to a bot account.")
-    parser.add_argument("-v", action="store_true", help="Make output more verbose. Include all communication with lichess.org.")
+    parser.add_argument("-v", action="store_true", help="Make output more verbose. Include all communication with lichess.org."
+                        )
```

Comment:
Because it was one character over the length limit, so there was no reason to split the string.

Reply:
I didn't know about the different maximum line length. On my computer, flake8 defaults to a maximum line length of 79 characters.

Just for aesthetic reasons, I think deleting the `.org` from `lichess.org` would be a better way to shrink the line. The hanging parentheses looks weird to me. Everything else in this PR is fine.

Reply:
The limit is specified in `python-build.yml` and it has been 127 since the file was created (2faa85b1a5defb8f915a6890fb4e0e78bb529cc5). The default is 79 from [PEP 8](https://peps.python.org/pep-0008/#maximum-line-length). When I run `flake8` on my computer, I copy the line from `python-build.yml` to make sure that I am running with the correct settings.

Reply:
Got it. Thanks for pointing that out.